### ISSUE TITLE
Placard for other controls

### ISSUE THREAD

**Issue by filiperinaldi** [ExactTarget/fuelux#1392]
Can the component Placard be adapted for other controls as well (selectlist, pillbox, checklist, etc)?
This would make FuelUX a very good front-end for inline editing.

**Comment by interactivellama** [comment]
You mean like put put a selectlist inside a placard instead of a text area?

**Comment by filiperinaldi** [comment]
Exactly.

**Comment by interactivellama** [comment]
I haven't tried it, but I think it could probably be anything that manipulated an input with the `.placard-field` class. 

You can also specify an `onAccept` and `onCancel` callback that will execute when ok and cancel buttons are pressed. Try getting the selected value from a selectlist and placing the value in a hidden input with `.placard-field`. Using `placard('setValue')` in your callback might work, too.

**Comment by cormacmccarthy** [comment]
We will be addressing this in an upcoming release. Thank you!

**Comment by cirovladimir** [comment]
I was trying to display a select list -in angular- when the placard is shown. I achieved it with the following code. I had to put the select list inside the placard-popup div and give it a margin so it didn't overlap with the header. I don't know if that's what has been addressed since version 3.11.4 and that's the purpose of the placard-popup div, it doesn't say in the docs or at least i couldn't found it.

```
<div id="tipoAudiencia" class="placard" data-initialize="placard" >
    <div class="placard-popup">
        <div class="form-group" style="margin-top: 30px;">
            <select class="form-control placard-field" id="slctTipoAudiencia"
                    ng-model="tipoAudienciaEditada"
                    ng-options="tipoAudiencia as tipoAudiencia.nombre for tipoAudiencia in tiposAudiencia 
                | orderBy: 'nombre' track by tipoAudiencia.id"
                    required>
                <option value=""></option>
            </select>
        </div>
    </div>
    <div class="placard-header">Selecciona el tipo de audiencia</div>
    <div contenteditable class="form-control placard-field glass">{{audiencia.tipoAudiencia.nombre}}</div>
    <div class="placard-footer">
        <a class="placard-cancel" href="#">Cancelar</a>
        <button class="btn btn-primary btn-xs placard-accept" type="button">Aceptar</button>
    </div>
</div>
```

then in my controller I update the value if it gets accepted

```
$('#tipoAudiencia').on('accepted.fu.placard', function(){
    console.log('actualizando el tipo de audiencia a ' + $scope.tipoAudienciaEditada);
    $scope.audiencia.tipoAudiencia = $scope.tipoAudienciaEditada;
    Audiencia.update($scope.audiencia);
});
```
![peek 2017-11-23 08-51](https://user-images.githubusercontent.com/2171465/33178484-ed11b0ca-d02b-11e7-86db-7ccf150925ef.gif)

I had some minor issues with sizing but they're not showstoppers

![peek 2017-11-23 08-52](https://user-images.githubusercontent.com/2171465/33178592-5757875c-d02c-11e7-9a2d-79e9138cbd47.gif)
